### ISSUE TITLE
Imag Preview in Image node is tiny

### ISSUE THREAD

**Issue by MaximumBlast** [comfyanonymous/ComfyUI#565]
Is it just me? If i load an image it is displayed super tiny....

![tiny](https://user-images.githubusercontent.com/60583831/234121944-d8eba3a3-5a9e-463c-8fda-86a8fca2f23f.png)

**Comment by WASasquatch** [comment]
It's not resizing the node, I swear it did this in older versions. But you can manually resize the node to show the image at any size.

**Comment by DivinoAG** [comment]
I've only been using Comfy for a few weeks and it's always been like this to me. But you're right, you can just resize the node to fix it.

**Comment by WASasquatch** [comment]
> I've only been using Comfy for a few weeks and it's always been like this to me. But you're right, you can just resize the node to fix it.

In any case, I wonder if that's hard to add. It is a little finicky trying to grab those corners to resize sometimes. Lol

**Comment by comfyanonymous** [comment]
Should be fixed now.

**Comment by MaximumBlast** [comment]
Yes, great! Thank you! Also found out how to resize nodes, Kind of hard to find. Thanks for the patch!

**Comment by ghostsquad** [comment]
> Yes, great! Thank you! Also found out how to resize nodes, Kind of hard to find. Thanks for the patch!

How do you resize nodes?

**Comment by WASasquatch** [comment]
> > Yes, great! Thank you! Also found out how to resize nodes, Kind of hard to find. Thanks for the patch!
> 
> How do you resize nodes?

Grab the bottom right corner, perhaps?

**Comment by Woisek** [comment]
> > How do you resize nodes?
> 
> Grab the bottom right corner, perhaps? The trigger area is only like a pixel big, so it's a challenge.

Yeah ... I made some suggestions here -> https://github.com/comfyanonymous/ComfyUI/issues/1697
